api/parts/data/fetch.js

```diff
@@ -685,6 +685,18 @@ fetch.fetchLoyalty = function(params) {
     });
 };
 
+/**
+* Get session data and output to browser
+* @param {string} collection -collection name
+* @param {params} params - params object
+* @param {boolean} isCustomEvent - if is custom event
+* @param {object} options - options object
+* @param {function} callback - calback function
+**/
+fetch.fetchViews = function(collection, params, isCustomEvent, options, callback) {
+    fetchTimeObj(collection, params, isCustomEvent, options, callback);
+};
+
```

Comment:
Let's not add plugins related methods in core
better use this:
https://github.com/Countly/countly-server/blob/master/api/parts/data/fetch.js#L915-L919